doc/docs/master/how-tos/basic-data-operations/sql-ingest.md

```diff
@@ -192,3 +193,58 @@ You have run a query using SQL Ingest. How do you inspect its result?
     {"mycolumn":"hello world","id":1}
     {"mycolumn":"hello you","id":2}
     ```
+
+## Formats and SQL DataTypes 
+
+The following comments on formatting reflect the state of this release and are subject to change.
+
+### SQL datatypes supported
+
+We support the following SQL datatypes. Some of those Data Types are specific to databases.
+
+| Dates/Timestamps | Varchars | Numerics | Booleans |
+|------------------|---------|----------|----------|
+|`DATE` <br> `TIME`<br> `TIMESTAMP`<br> `TIMESTAMP_LTZ`<br> `TIMESTAMP_NTZ`<br> `TIMESTAMP_TZ`<br> `TIMESTAMPTZ`<br> `TIMESTAMP WITH TIME ZONE`<br> `TIMESTAMP WITHOUT TIME ZONE` |`VARCHAR`<br> `TEXT`<br> `CHARACTER VARYING`|`SMALLINT`<br> `INT2`<br> `INTEGER`<br> `INT`<br> `INT4`<br> `BIGINT`<br> `INT8`<br>`FLOAT`<br> `FLOAT4`<br> `FLOAT8`<br> `REAL`<br> `DOUBLE PRECISION`<br>`NUMERIC`<br> `DECIMAL`<br> `NUMBER`<br> `FIXED`|`BOOL`<br>`BOOLEAN`|
```

Comment:
`FIXED` is not a database type, it is a type specific to Snowflake returned by the Go driver.